setup.cfg

```diff
@@ -93,5 +93,9 @@ warn_return_any = true
 warn_unused_configs = true
 warn_unused_ignores = true
 
+# Avoid error: Duplicate module named 'setup'
+# https://github.com/python/mypy/issues/4008
+exclude = ^tests/test_data/
```

Comment:
I think we should update the invocation instead.

Reply:
I will look into addressing this in a follow-up as this fix is unrelated to current change (the exclude was already there).  I am sure we will have more mypy work done soon, but now I need to unblock the CI/CD gates.